ontopy/ontology.py

```diff
@@ -1746,12 +1798,14 @@ def new_entity(
 
         with self:
             entity = types.new_class(name, parents)
+
+            # AREN'T WE LITTLE TOO SMART HERE?
```

Comment:
Why? The idea is that if we make a new entity it is given prefLabel immediately. This can be changed of course afterwards, but generally I think this would be normal behaviour, no?

Reply:
updated as agreed